ee/localserver/server.go

```diff
@@ -62,28 +61,16 @@ const (
 	defaultRateBurst = 10
 )
 
-type LocalServerOption func(*localServer)
-
-func WithLogger(logger log.Logger) LocalServerOption {
-	return func(s *localServer) {
-		s.logger = log.With(logger, "component", "localserver")
-	}
-}
-
-func New(k types.Knapsack, opts ...LocalServerOption) (*localServer, error) {
+func New(k types.Knapsack) (*localServer, error) {
 	ls := &localServer{
-		logger:                log.NewNopLogger(),
+		logger:                k.Slogger().With("component", "localserver"),
```

Comment:
I ight have called this `slogger`, but okay!

Reply:
using slogger